packages/fhir-converter/src/templates/cda/Header.hbs

```diff
@@ -23,7 +23,14 @@
             {{>Resources/Organization.hbs org=this.custodian.assignedCustodian.representedCustodianOrganization ID=(generateUUID (toJsonString this.custodian.assignedCustodian.representedCustodianOrganization))}},
     		{{>References/Composition/custodian.hbs ID=(generateUUID (toJsonString this)) REF=(concat 'Organization/' (generateUUID (toJsonString this.custodian.assignedCustodian.representedCustodianOrganization)))}},            
         {{/if}}
-        
+
+        {{#each participant}}
+            {{#if this}}
+                {{>Resources/RelatedPerson.hbs relatedPerson=this.associatedEntity ID=(generateUUID (toJsonString this))}},
+                {{>References/Patient/link.other.hbs ID=(generateUUID (toJsonString this)) REF=(concat 'RelatedPerson/' (generateUUID (toJsonString this.associatedEntity)))}},
+            {{/if}}
+        {{/each}}
+
```

Comment:
@RamilGaripov is the `RelatedPerson` ID is being generated from `this`, shouldn't the `'RelatedPerson/' (generateUUID (toJsonString this.associatedEntity` be `'RelatedPerson/' (generateUUID (toJsonString this` instead?

Reply:
good catch